src/service/dealService.ts

```diff
@@ -280,6 +282,71 @@ const createCoupangImage = async (req, res, next) => {
   }
 };
 
+const readDealDetail = async (req, res, next) => {
+  try {
+    const dealId: number = +req.params.dealId;
+    const userId: number = +req.query.userId;
+    const deal = await dealRepository.getDealDetail(dealId);
+    const group = await groupRepository.findGroupByUserIdAndDealId(
+      userId,
+      dealId,
+    );
+
+    const userStatus = await dealModule._checkUserStatusInDeal(
+      group,
+      userId,
+      dealId,
+    );
+
+    const dealWithStatusDto: DealWithStatusDto = new DealWithStatusDto(deal);
+    dealWithStatusDto['mystatus'] = userStatus.description;
+    dealModule._setDealStatus(dealWithStatusDto);
+
+    success(res, statusCode.OK, responseMessage.SUCCESS, dealWithStatusDto);
+  } catch (error) {
+    logger.error(error);
+    next(error);
+  }
+};
+
+const homeAllDeal = async (req, res, next) => {
```

Comment:
```suggestion
const homeAllDeal = async (req: Request, res: Response, next: NextFunction) => {
```
과 같이 타입을 지정하는게 어떨까요?